developer/src/test/auto/kmcomp/Makefile

```diff
@@ -1,13 +1,13 @@
 #
-# Test that kmcomp passes various compile tests
+# Test that kmc passes various compile tests
 #
-# Uses kmcomp.exe rather than attempting to run a separate test
+# Uses kmc.exe rather than attempting to run a separate test
```

Comment:
Is there a `kmc.exe`? Below it calls `kmc.cmd`.

Reply:
```suggestion
# Uses kmc.js (via kmc.cmd) rather than attempting to run a separate test
```